docs/source/model_doc/roberta.rst

```diff
@@ -44,7 +44,8 @@ Tips:
   separate your segments with the separation token :obj:`tokenizer.sep_token` (or :obj:`</s>`)
 - :doc:`CamemBERT <camembert>` is a wrapper around RoBERTa. Refer to this page for usage examples.
 
-The original code can be found `here <https://github.com/pytorch/fairseq/tree/master/examples/roberta>`_.
+This model was contributed by `thomwolf <https://huggingface.co/thomwolf>`__. The original code can be found `here
```

Comment:
I believe this was @julien-c

Reply:
Yes! my first (and only) model ;)

You can also attribute `@myleott`?